lib/matplotlib/mathtext.py

```diff
@@ -842,6 +842,11 @@ def _get_glyph(self, fontname, font_class, sym, fontsize, math=True):
             found_symbol = False
             font = self._get_font(new_fontname)
             if font is not None:
+                if font.family_name == "cmr10" and uniindex == 0x2212:
```

Comment:
```suggestion
                if uniindex == 0x2212 and font.family_name == "cmr10":
```
I'd switch the conditions. The character is more selective than the font and that makes the scope slightly easier to understand.